### ISSUE TITLE
`accessibility.verboseChatProgressUpdates` does not read CoT

### ISSUE THREAD

**Issue by anthonykim1** [microsoft/vscode#269758]
When `accessibility.verboseChatProgressUpdates` is enabled, it would not read CoT. 

/ cc @jooyoungseo and @rperez030

**Comment by meganrogge** [comment]
@rperez030 I noticed that when I log what we're alerting, all the messages are present. But when I enable the screen reader, only some are read. I believe this is a timing issue. Using `status` vs `alert` did not fix it. Any suggestions? Not sure if there's anything we can do about it.

**Comment by rperez030** [comment]
> [@rperez030](https://github.com/rperez030) I noticed that when I log what we're alerting, all the messages are present. But when I enable the screen reader, only some are read. I believe this is a timing issue. Using `status` vs `alert` did not fix it. Any suggestions? Not sure if there's anything we can do about it.

The problem with alerts is that they override each other if they are too close in time, or even if they are being announced when a new one arrives in the case of VoiceOver. 

**Status** should be the role for progress messages unless they are time-sensitive, such as confirmations or errors. 

I believe the logic uses a 4-second interval, so I don't think timing is the issue, but maybe increasing it to 5 seconds would help. 

Is this using the same existing infrastructure that is used for chat responses?
It is also interesting that I still hear the "working" message, which I thought had been replaced by the new progress messages.

**Comment by meganrogge** [comment]
We still use the `working` message because extension contributed tool messages aren't read out currently. https://github.com/microsoft/vscode/issues/266256

I tried `status`, which led to nothing being announced 🫠 

Maybe if we only say `working` when these verbose ones are disabled, and set the chat response progress announcement to off by default, then the `status` will come through?

**Comment by meganrogge** [comment]
Using this to track the issue when using Codex as the model and not hearing announcements about the thinking progress

**Comment by justschen** [comment]
i believe this was fixed by @meganrogge ?
